src/polyfills/details/_base.scss

```diff
@@ -56,48 +48,12 @@ details {
 			}
 		}
 	}
-
-	// Prevent FOUC
-	&:not([open]) {
-		@extend %details-nested-in-hidden-details;
-		visibility: hidden;
-
-		> {
-			// Need details and * to deal with specificity issues
-			details,
-			* {
-				display: none;
-			}
-		}
-	}
-
-	&.alert {
-		&:not([open]) {
-			visibility: visible;
-		}
-	}
-
-	.out {
-		@extend %details-nested-in-hidden-details;
-	}
-}
-
-.tabpanels {
-	> {
-		details {
-			&:not([open]) {
-				visibility: visible;
-			}
-		}
-	}
 }
 
 // Prevent FOUC when polyfill is disabled
 .wb-disable {
```

Comment:
@Christopher-O Could you also remove this whole rule? I just tested in basic mode and doesn't seem to help with anything. It actually causes issues for table elements and other elements that are not by default block elements.

Reply:
@Garneauma  just to confirm, you're suggesting to remove the entire rule for .wb-disable?

Reply:
@Christopher-O Yes, exactly. Thank you.